src/main/java/org/radarcns/redcap/managementportal/Subject.java

```diff
@@ -57,7 +57,7 @@ public Subject(
             @JsonProperty("externalId") Integer externalId,
             @JsonProperty("externalLink") URL externalLink,
             @JsonProperty("project") Project project,
-            @JsonProperty("attributes") List<Tag> attributes) {
+            @JsonProperty("attributes") Attribute attributes) {
```

Comment:
I think this should be a `Map<String, String>`. The data model in MP allows the attributes to be any key-value pair where both key and value are strings. The UI currently only allows `Human-readable-identifier` as a key, but the intent is to have more keys in the future. Perhaps even allow the users to create their own keys, still needs to be discussed. But to have this as general as possible, a `Map<String, String>` would be better here.